Browser_IDE/executionEnvironment.html

```diff
@@ -6,24 +6,27 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SplashKit Runtime</title>
-
     <link rel="stylesheet" href="stylesheet.css">
+    <script src="https://cdn.jsdelivr.net/npm/split.js@1.6.2"></script>
 </head>
 
 <body class="sk-body">
-	<div class="sk-column" style="max-height: 100%;">
-		<div class="sk-contents" style="flex-direction: row; max-height: 85%;">
-			<canvas id="canvas" style="margin:auto; vertical-align:center; max-height: 100%; left:0px; right:0px; width:inherit; outline: none;" oncontextmenu="event.preventDefault()" tabindex=-1></canvas>
-		</div>
-		<div class="sk-header sk-header-indent">
-			TERMINAL OUTPUT
-			<button id="clearButton" class="btn btn-outline-secondary btn-sm ms-2" onclick="clearTerminal()">Clear</button>
-		</div>
-		<div class="sk-contents">
-			<div id="output" class="sk-contents sk-contents-darker sk-terminal"></div>
-		</div>
-	</div>
+    <div class="sk-column" style="height: 100%; display: flex; flex-direction: column;">
+        <div id="canvasContainer" style="flex-grow: 1; display: flex; position: relative;">
+            <canvas id="canvas" style="position: absolute; margin: auto; outline: none;" oncontextmenu="event.preventDefault()" tabindex="-1"></canvas>
```

Comment:
Remove `position: absolute`. It is all that prevents you from using `margin: auto` to center the canvas rather than manually calculating the margin values.